charts/scalardb/values.yaml

```diff
@@ -0,0 +1,108 @@
+# Default values for scalardb.
```

Comment:
Please create the custom values as well. (maybe in scalar-kubernetes repo)
ref. https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/scalardl-custom-values.yaml

Reply:
I'll create a PR for that after finishing the test including monitoring with prometheus